src/SideAppbar/AppLogo.js

```diff
@@ -0,0 +1,34 @@
VjMi45NCwwLDUuNy0uNjIsOC4zLTEuODYsMi42LTEuMjQsNC44NS0yLjkyLDYuNzctNS4wMiwxLjkxLTIuMTEsMy40My00LjU5LDQuNTYtNy40NSwxLjEzLTIuODYsMS42OS01LjkxLDEuNjktOS4xNXYtMzEuMzljMC0zLjI0LS41Ny02LjI4LTEuNjktOS4xNFptLTEyLjU2LDQwLjUzYzAsMi4xOC0uNjgsNC4wMy0yLjA1LDUuNTMtMS4zNywxLjUxLTMuMDQsMi4yNi01LjAyLDIuMjZoLTM1LjVjLTEuOTgsMC0zLjY2LS43NS01LjAyLTIuMjYtMS4zNy0xLjUtMi4wNS0zLjM1LTIuMDUtNS41M3YtMzEuMzljMC0yLjE4LjY4LTQuMDMsMi4wNS01LjUzczMuMDQtMi4yNiw1LjAyLTIuMjZoMzUuNWMxLjk4LDAsMy42NS43NSw1LjAyLDIuMjZzMi4wNSwzLjM1LDIuMDUsNS41M3YzMS4zOWgwWiIvPjxwYXRoIGNsYXNzPSJ1dWlkLWY4NDVjMTY2LTViMWMtNDE4Ni1iOTA3LTc5ZjA1ZWY0ZjQxOCIgZD0ibTE4OC45OCw2LjkzYy0xLjk1LTIuMTEtNC4yNS0zLjc4LTYuOS01LjAyUzE3Ni42LjA1LDE3My42MS4wNWgtNTMuNDdjLTEuNDYsMC0yLjQzLDEuNTEtMS44MiwyLjg0bDUuMzgsMTEuNjljLjMzLjcxLDEuMDQsMS4xNiwxLjgyLDEuMTZoNDguMDljMi4wMiwwLDMuNzMuNzUsNS4xMywyLjI2LDEuMzksMS41MSwyLjA5LDMuMzUsMi4wOSw1LjUzcy0uNyw0LjA1LTIuMDksNS41OWMtMS40LDEuNTQtMy4xLDIuMzEtNS4xMywyLjMxaC0zNC44M2MtMywwLTUuODIuNjItOC40NywxLjg2LTIuNjUsMS4yNC00Ljk1LDIuOTItNi45LDUuMDItMS45NSwyLjExLTMuNSw0LjU5LTQuNjUsNy40NS0xLjE1LDIuODYtMS43Myw1LjkxLTEuNzMsOS4xNXYyMS40OGMwLDEuMTEuOSwyLDIsMmgxMC41NGMxLjEsMCwyLS45LDItMnYtMjEuNDhjMC0yLjE4LjctNC4wMywyLjA5LTUuNTNzMy4xLTIuMjYsNS4xMy0yLjI2aDE1LjU5Yy42NSwwLDEuMjcuMzIsMS42NC44NmwyMC41NiwyOS41NmMuMzcuNTQuOTkuODYsMS42NC44NmgxMy4zM2MxLjYyLDAsMi41Ni0xLjgyLDEuNjQtMy4xNWwtMTkuNTctMjguMTNjMywwLDUuODItLjYyLDguNDctMS44NnM0Ljk1LTIuOTQsNi45LTUuMDhjMS45NS0yLjE1LDMuNS00LjY1LDQuNjUtNy41MSwxLjE1LTIuODYsMS43My01LjkxLDEuNzMtOS4xNXMtLjU4LTYuMjgtMS43My05LjE1Yy0xLjE1LTIuODYtMi43LTUuMzQtNC42NS03LjQ1aDBaIi8+PHBhdGggY2xhc3M9InV1aWQtZjg0NWMxNjYtNWIxYy00MTg2LWI5MDctNzlmMDVlZjRmNDE4IiBkPSJtOTEuMy4wMmgxMC41NGMxLjExLDAsMiwuOSwyLDJ2NzQuMzdjMCwxLjExLS45LDItMiwyaC0xMC41NGMtMS4xLDAtMi0uOS0yLTJWMi4wMWMwLTEuMS45LTIsMi0yaDBaIi8+PHBhdGggY2xhc3M9InV1aWQtZjg0NWMxNjYtNWIxYy00MTg2LWI5MDctNzlmMDVlZjRmNDE4IiBkPSJtNzMuOTcuMDJoLTEwLjY0Yy0xLjExLDAtMi4wMi44My0yLjAyLDEuODV2MTIuNjNjMCw1LjMxLS4zLDkuOTYtLjksMTMuOTgtLjA1LjMzLS4xLjY4LS4xNSwxLjAxLS43LDQuMjMtMS45NCw4LjA4LTMuNjksMTEuNTQtMS43NywzLjQ4LTQuMTYsNi43OC03LjE5LDkuOS0yLjU4LDIuNjctNC4zNyw1LjA3LTkuMDcsNy43MS0uNTUuMzEtMS4xNC41LTEuNzUuNTgtLjE5LjAzLS4zOC4wNC0uNTYuMDQtLjE5LDAtLjM4LS4wMS0uNTctLjA0LS42MS0uMDgtMS4yLS4yNy0xLjc1LS41OC00LjctMi42NC02LjQ5LTUuMDQtOS4wNy03LjcxLTMuMDMtMy4xMi01LjQyLTYuNDItNy4xOS05LjktMS43NS0zLjQ2LTIuOTktNy4zMS0zLjY5LTExLjU0LS4wNS0uMzMtLjEtLjY4LS4xNS0xLjAxLS42LTQuMDItLjktOC42Ny0uOS0xMy45OFYxLjg3QzE0LjY4Ljg1LDEzLjc3LjAyLDEyLjY2LjAySDIuMDJDLjkuMDIsMCwuODUsMCwxLjg3djguOThjMCw3LjI5LjM1LDEzLjczLDEuMDUsMTkuMzEuNyw1LjU5LDIuMTIsMTAuNzQsNC4yNCwxNS40NiwyLjExLDQuNzIsNS4wOSw5LjIsOC45MywxMy40MywyLjE4LDIuMzksNC44NCw1LjAyLDcuNzksNy43NCwzLjM1LDMuMDksNi45Miw1Ljk0LDEwLjYyLDguNTlsMy4wNiwyLjE5YzEuNDMsMS4xLDMuMTcsMS4xLDQuNjEsMGwzLjA2LTIuMTljMy43LTIuNjUsNy4yNy01LjUsMTAuNjItOC41OSwyLjk1LTIuNzIsNS42MS01LjM1LDcuNzktNy43NCwzLjg0LTQuMjMsNi44Mi04LjcxLDguOTMtMTMuNDMsMi4xMi00LjcyLDMuNTQtOS44Nyw0LjI0LTE1LjQ2LjctNS41OCwxLjA1LTEyLjAyLDEuMDUtMTkuMzFWMS44N2MwLTEuMDItLjktMS44NS0yLjAyLTEuODVaIi8+PC9nPjwvZz48L3N2Zz4=";
```

Comment:
General comment about icons. We should really have a separate component per icon, and rather than embed data:image strings, we should export them from Figma as SVG, and expose them as a MUI SvgIcon. e.g. this would become something like:

https://codesandbox.io/s/new-thunder-bj1sfp?file=/demo.tsx

This applies to the Virto icon, but also all the individual app icons. If we expose these through react-ui, then they can be reused in each app, as well as the app-store where they are currently hardcoded. If we ever want to change an icon, we then only have to do it in one place.